$NetBSD$

Add sun.c to the source list when Sun audio is enabled

--- Makefile.am.orig	2018-03-14 09:47:45.000000000 +0000
+++ Makefile.am
@@ -90,3 +90,9 @@ EXTRA_DIST += \
 	src/TPCircularBuffer/README.markdown \
 	src/TPCircularBuffer/TPCircularBuffer.podspec
 endif
+
+# Sun audio support
+if HAVE_SUN
+src_libpcaudio_la_SOURCES += \
+	src/sun.c
+endif
